common/persistence/sql/sqlplugin/queue_v2_metadata.go

```diff
@@ -22,24 +22,40 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package sql_test
+package sqlplugin
 
 import (
 	"context"
-	"testing"
+	"database/sql"
 
-	"github.com/stretchr/testify/assert"
-	"go.temporal.io/server/common/persistence/sql"
+	"go.temporal.io/server/common/persistence"
 )
 
-func TestNewQueueV2(t *testing.T) {
-	t.Parallel()
+type (
+	// QueueMetadataRow represents a row in queue_metadata table
```

Comment:
Should be QueueV2MetadataRow

Reply:
Done

Reply:
This still shows up incorrectly for me

Reply:
It looks correct in the "Files changed" tab now.